docs/staking.md

```diff
@@ -0,0 +1,48 @@
+---
+id: staking
+title: Staking
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+HydraDX has a **long-running HDX staking program** which **incentivizes user activity** in areas that are beneficial to the Protocol. On this page you will find important information regarding the **mechanics behind the HDX Staking program**. You can also check out our [step-by-step guide on staking](/howto_stake).
+
+## Staking Basics {#basics}
+
+HDX holders can **stake their HDX and receive rewards** which become **claimable as time passes**. Staking rewards are distributed from a dedicated pot that is gradually filled up by **different Protocol revenue streams**. Initially, the main revenue stream are the **LP fees** which the HydraDX Protocol accrues from its **HDX LP position in the Omnipool**. Furthermore, there is a proposal to subsidise APR during the first year of the staking program by **allocating an additional subsidy of ~22M HDX** from the HydraDX Treasury.
+
+The pot with staking rewards is periodically distributed in its fullest to all stakers. The amount that users are entitled to is proportional to the relative size of their stake in the stake pool. However, stakers do not automatically receive the rewards on their account - instead, they need to claim them.
```

Comment:
> The pot with staking rewards is periodically distributed...
Rewards are not distributed "periodically", they are distributed as soon as they hit the pot. Calculation will happen async, on user's actions, but pot is basically always distributed/empty, if any staking position exists. 
Only time when pot is not distributed is when pending rewards in the pot are to small to distribute